src/clickup-cdk.ts

```diff
@@ -61,6 +62,16 @@ export module clickupCdk {
      * @default undefined
      */
     readonly sendReleaseEventOpts?: datadog.ReleaseEventOptions;
+
+    /**
+     * Should we send a slack webhook on release (required for compliance audits)
+     */
+    readonly sendSlackWebhookOnRelease?: boolean;
```

Comment:
Since this is an optional, you need to define the default.

Also, consider renaming the param so that it can default to `false` (because both false and undefined are falsy, so your code ends up cleaner)

```suggestion

    /**
     * Should we send a slack webhook on release (required for compliance audits)
     *
     * @default true
     */
    readonly sendSlackWebhookOnRelease?: boolean;
```